xcp_d/interfaces/resting_state.py

```diff
@@ -6,8 +6,8 @@
 
 """
 import os
-import shutil
 
+import shutil
```

Comment:
`shutil` is a builtin module, so it should go up with `os`, not with the third-party modules like `nilearn` and `nipype`.